tests/marqo_test.py

```diff
@@ -236,8 +255,12 @@ def create_open_source_index(self, index_name: str, settings_dict: dict = None,
         client = marqo.Client(**self.client_settings)
         if settings_dict and kwargs:
```

Comment:
What if settings_dict is an empty dict? Isn't this valid settings_dict value? 

Can we be more explicit to prevent against related edge cases? ?